doc/rose-rug-advanced-tutorials-suicide.html

```diff
@@ -260,6 +260,282 @@ <h3 class="alwayshidden">Note on suicide triggers (2)</h3>
         <li>triggering off a cleanup task in the suite</li>
       </ul>
     </div>
+
+    <div class="slide">
+          <h2 id="advanced-example">Advanced suicide triggers example</h2>
+      <p>The following example is one suite but with three possible 
+      scenarios:</p>
+
+      <dl>
+        <dd>If a member of a family fails carry on.
```

Comment:
Close, see e.g. rose-api.html for an example of what I intended. Apologies for lack of clarity on my part.